Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/RotationManipulators.cpp

```diff
@@ -158,4 +158,22 @@ namespace AzToolsFramework
     {
         m_circleBoundWidth = circleBoundWidth;
     }
+
+    AZStd::vector<BaseManipulator*> RotationManipulators::GetManipulators()
+    {
+        AZStd::vector<BaseManipulator*> manipualators;
+        for (auto& manipulator : m_localAngularManipulators)
+        {
+            if (manipulator)
+            {
+                manipualators.emplace_back(manipulator.get());
+            }
+        }
+        if (m_viewAngularManipulator)
+        {
+            manipualators.emplace_back(m_viewAngularManipulator.get());
+        }
+        return manipualators;
+    }
+
```

Comment:
update: Adding this might not be the best way to achieve what we want, there could be an alternative design that does not require this. If you could give some more context on why this is needed and what you're trying to achieve I'd be very grateful

Reply:
There is a typo in the "manipualators" variable. It should be "manipulators".

Normally when I need to expose a container to client code, instead of returning a `vector`, `set` or some other container type, I instead prefer to provide a visitor function with a callback that the client can use to populate their own container

Such as 
```c++
using ManipulatorVisitCallback = AZStd::function<void(const AZStd::shared_ptr<AngularManipulator>&)>;
// ...
void RotationManipulators::VisitManipulators(const ManipulatorVisitCallback& callback) const
{
    for (const auto& manipulator : m_localAngularManipulators)
    {
        if (manipulator)
        {
           callback(manipulator);
        }
    }
    if (m_viewAngularManipulator)
    {
        callback(m_viewAngularManipulator);
    }
}
```

That way if you the client wanted to populate a vector or any other container type they can just a pass in a lambda.

```c++
AZStd::vector<BaseManipulator*> baseManipulators;
auto GetBaseManipulators = [&baseManipulator](const AZStd::shared_ptr<AngularManipulator>& angularManipulator)
{
    baseManipulators.push_back(angularManipulator.get());
};
rotationManipulators.VisitManipulators(GetBaseManipulators);
// Use the baseManipulators array as needed.
```